### DISCUSSION TITLE
Never show me anything "trending" ever for any reason

### DISCUSSION THREAD

**Discussion by mcclure** [community/community#68430] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Some months ago, the front page feed split into the "real feed" containing information relevant to me, and a "for you" feed containing stupid [several curse words removed here] your [curse word] "algorithms" want me to look at for I don't know what reason. I and many others requested in this feedback forum that the "for you" feed be removed, because I don't want it as a feature. Now instead you have done the opposite, merging the "for you" feed into the "real" feed, so now I *cannot turn it off*.

![image](https://github.com/community/community/assets/277318/782347db-e5bc-4b97-9adb-32c32643f2bb)

You claim there's filtering, but there is no way to filter "For You" content. "For You" content has a … next to which has a menu option I can use to "see less", which if I click it allows me to select an option I want to see "less trending content". That's not what I want. I don't want to see "less" irrelevant algorithmic content. I never want to see algorithmic content at all. This is abuser logic. I don't want it, so you allow me to give the meaningless feedback that I want to see it "less".

There's an "ask for feedback", but you are not reading this feedback, or if someone is reading this feedback it's a community relations person who will be ignored by anyone who makes a decision.

All you are doing is assigning me work. Changing the interface every few months creates work for me because I must relearn it. Making me click "see less" every time you show something that you already know I don't want to see (and, most likely, no one at all wants to see) is assigning me work. Asking for "feedback" you don't even read is assigning me work.

I hate this, and I hate Github now. I wish Microsoft had never bought Github.

**Comment by bitsnaps** [comment]
I really miss the "trending" feed feature, now with the LLMs racing we really need it

**Comment by Apreche** [comment]
100% agreement. In the feed I want to see exactly what I want to see, in reverse chronological order, and nothing else at all. No suggestions, no recommendations, no trends, no nonsense.
